deployment/terraform/assets/variables.tf

```diff
@@ -60,8 +60,8 @@ variable "db_instance_class" {
 variable "db_engine_version" {
   type = map(any)
   default = {
-    "aurora-mysql"      = "5.7.mysql_aurora.2.10.0"
-    "aurora-postgresql" = "11.7"
```

Comment:
There was a reason we used this specific version as it's what Cloud adopted at the time. It would be interesting to know what they have landed on as of late.

Reply:
@stylianosrigas what version are you guys on in cloud these days?

Reply:
We are on 12.8 for PostgreSQL.

Reply:
Ok, I'll merge this as is for now but it might be worth exploring bumping to 12.8 to match cloud.